### PR TITLE
feat: added docs for meeting transcripts

### PR DESCRIPTION
**Pull request by roerohan**
## Description

This PR adds the documentation for meeting AI - transcription.

## Resolved issues

NA

### Before submitting the PR, please take the following into consideration
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. If you don't have an issue, please create one.
- [x] Prefix your PR title with `feat: `, `fix: `, `chore: `, `docs:`, or `refactor:`.
- [x] The description should clearly illustrate what problems it solves.
- [x] Ensure that the commit messages follow our guidelines.
- [x] Resolve merge conflicts (if any).
- [x] Make sure that the current branch is upto date with the `main` branch.

### CODE REVIEW

File: docs/guides/capabilities/ai/meeting-transcription.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Dyte's meeting transcription allows you to transcribe your Dyte meetings in real-time, making it easy to capture important discussions and refer back to them later.
+This guide will walk you through how to use this feature effectively.
+
+## Control transcriptions for participants using presets
+
+You can control whether or not a participant's audio will be transcribe with the help of the `transcription_enabled` flag in the participant's preset.
+All participants with the `transcription_enabled` turned on in their preset will be able to generate transcripts in real-time in a Dyte meeting.
+
+You can create a new preset on our [Developer Portal](https://dev.dyte.io/presets), or using our [REST API](https://docs.dyte.io/api#/operations/post-presets)
```
Comment (manyaagarwal) [inline docs/guides/capabilities/ai/meeting-transcription.mdx:15]: Here we could redirect the user to the Presets guide through,

> To learn more about how to create and edit presets, see [Add Presets](/guides/live-video/concepts-live-video)

File: docs/guides/capabilities/ai/meeting-transcription.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Consuming transcripts
+
+There are 2 ways in which these transcripts can be consumed.
```
Comment (manyaagarwal) [inline docs/guides/capabilities/ai/meeting-transcription.mdx:19]: For the bullet points, the first words can be used to indicate the way in which the transcript can be consumed to make it easy to glance through. Something like: 

> 1. Client coreSDK : The transcripts can be consumed on the client-side using the Dyte SDK that's suitable for your platform. These transcripts are generated on the server in real-time. 

>2. Webhooks: The meeting transcript can be consumed via a [webhook after the meeting ends](#consume-transcript-via-a-post-meeting-webhook).

File: docs/guides/capabilities/ai/meeting-transcription.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+## Testing transcription
+
+Once you have configured a preset and a webhook according to the instructions above, you can proceed to test whether meeting transcription is working for your organization.
+To test if meeting transcription has been configured for your organization, perform the following steps.
+
+1. [Create a meeting](https://docs.dyte.io/api#/operations/create_meeting).
+2. [Add a participant](https://docs.dyte.io/api#/operations/add_participant) to the meeting. Make sure that the preset you use was configured according to this guide.
+3. Join the meeting with the `authToken` you just obtained. As you unmute and speak (in English), your speech should be getting transcribed in real-time for all the participants in the meeting.
```
Comment (manyaagarwal) [inline docs/guides/capabilities/ai/meeting-transcription.mdx:51]: Could we add an attention/information box in the beginning that says that Dyte's transcription currently only supports English?

Comment (roerohan) [reply docs/guides/capabilities/ai/meeting-transcription.mdx:51]: image

Added this note after the introductory paragraph.

**Pull request merged**